source/schema.txt

```diff
@@ -55,9 +55,6 @@ percentage of each data type represented.
      that Compass can provide a graded bar of the field contents, with
      the most-frequently occurring string on the left.
 
-   - The ``categories`` field contains arrays of strings. The analysis
-     shows the minimum, maximum, and average array lengths.
-
```

Comment:
Compass still shows this information for arrays of strings. We probably shouldn't delete it from our docs, but it would be nice to tune this to the new example in the updated screenshots.

Reply:
I updated the text and the image accordingly!